app/src/main/java/edu/uc/willi6jd/mycaloriecounter/FoodActivity.kt

```diff
@@ -21,26 +21,26 @@ class FoodActivity : AppCompatActivity() {
         val listView = findViewById<ListView>(R.id.food_listview)
         listView.adapter = foodCustomerAdapter(this)
 
-        val btnCreate = findViewById(R.id.btnCreateFoodItem) as Button //Create food item button
+        val btnCreate = findViewById<Button>(R.id.btnCreateFoodItem) //Create food item button
```

Comment:
Use Kotlin synthetics here... you don't need to use findViewById anymore.  Synthetics are much cleaner.